src/db_queries/auth.ts

```diff
@@ -165,82 +169,96 @@ export class Auth {
       response.status(400).send({'message': 'Please enter a valid email address'});
       return ;
     }
-    
+    const loginQuery = 'SELECT * FROM users WHERE email = $1';
```

Comment:
I would say to leave the query string right above the "const { rows }: pg.QueryResult" line so that we can see it better. This applies to other query strings in the same PR.